zerver/lib/push_notifications.py

```diff
@@ -353,69 +354,61 @@ async def send_all_notifications() -> (
 
 
 #
-# Sending to GCM, for Android
+# Sending to FCM, for Android
```

Comment:
This is called FCM now, even though we're using the legacy API that used
    to be called GCM.

More specifically it looks like we migrated to FCM in 2019: 630481cb7a4ab0bb2c481c53a86759d28e567161

They changed the endpoint URL, to one at fcm.googleapis.com, but kept supporting basically the old GCM API. And apparently the "v1" FCM API was already offered then, but just didn't give us much reason to switch:

    push notif: Switch from GCM to FCM endpoint.
    
    This is the only server-side change required for the FCM migration!
    
    Optionally, at some point in the future we might choose to migrate
    to the new ("v1") API which FCM also offers.  Nothing revolutionary
    but there are some nice things about it:
      https://firebase.google.com/docs/cloud-messaging/migrate-v1